lib/tasks/sidekiq.rake

```diff
@@ -43,14 +43,29 @@ namespace :sidekiq do
         dest: RedisJobSet.new(redis: dest_redis),
         logger: logger,
         progress: progress,
-        )
+      )
 
       delayed_queue = SidekiqMover.new(
         source: SidekiqDelayedQueue.new(redis: source_redis),
-        dest:   SidekiqDelayedQueue.new(redis: dest_redis),
+        dest: SidekiqDelayedQueue.new(redis: dest_redis),
         logger: logger,
         progress: progress
-        )
+      )
+      # https://github.com/mperham/sidekiq/blob/63ee43353bd3b753beb0233f64865e658abeb1c3/lib/sidekiq/api.rb#L641
```

Comment:
Add a blank line before this.

Reply:
:+1:

Reply:
https://github.com/mperham/sidekiq/wiki/API#retries might be a better link?

Reply:
Not really, it doesn't actually mention the name of the redis queue which is what matters here, no?